### PR TITLE
Add field rations from CSE

### PR DESCRIPTION
**Pull request by thojkooi**
Heavily WIP

### CODE REVIEW

**Comment by Glowbal** [comment]
Tagging @PabstMirror since he was the one that originally ported it to an ACE working branch.

**Comment by PabstMirror** [comment]
`local _consumeTime = getNumber (_cfg >> QGVAR(consumeTime));`
Really shows how long this code has been sitting there :smile: 

I got a little stuck finding good status effects to apply to the player. I think ace adding a stamina system will be what we really need to finish this up right.

There still is some localization and cleanup that needs doing. But I believe most of the code should be working and all the actions working as intended.

File: addons/field_rations/CfgSounds.hpp [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class CfgSounds {
+    class cse_drinking {
+        name = "cse_drinking";
+        sound[] = {PATHTOF(sounds\drinkingSound.ogg),"db-1",1.0};
```
Comment (jonpas) [inline addons/field_rations/CfgSounds.hpp]: `QPATHTOF`

**Comment by jonpas** [comment]
How far from done is this?

**Comment by PabstMirror** [comment]
This is pretty close to being ready.
How this module should work is somewhat subjective, but it should be fairly close to how it was in CSE.

**Comment by jonpas** [comment]
Continued work in #114.

**Comment by Zumiii** [comment]
Any progress on this one? Would be a huge Improvement for all the realism clans out there. Especially for persistent scenarios.

**Comment by jonpas** [comment]
https://github.com/acemod/ACEX/pull/114

**Pull request closed without merging**